pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -137,7 +137,7 @@ spec:
           volumeMounts:
             - name: plugin-dir
               mountPath: /csi
-          imagePullPolicy: "IfNotPresent"
+          imagePullPolicy: {{ .ImagePullPolicy }}
```

Comment:
imagePullPolicy is not set for all containers please add this to missing containers as well, take care of this for cephfs,rbd and nfs yamls also

Reply:
Yes it's set for all containers

Reply:
can you please check is it set for `driver-registrar` container?

Reply:
> can you please check is it set for `driver-registrar` container?

Done